File: Beecrowd/1021 Notas e Moedas/main.py
notas = float(input())
n = int(notas)
moedas = int((notas - n) * 100)

if (moedas * 1000) % 10 == 9:
    moedas += 1

print("NOTAS:")
print("{} nota(s) de R$ 100.00".format(int(n / 100)))
aux = int(n % 100)

print("{} nota(s) de R$ 50.00".format(int(aux / 50)))
aux = int(aux % 50)

print("{} nota(s) de R$ 20.00".format(int(aux / 20)))
aux = int(aux % 20)

print("{} nota(s) de R$ 10.00".format(int(aux / 10)))
aux = int(aux % 10)

print("{} nota(s) de R$ 5.00".format(int(aux / 5)))
aux = int(aux % 5)

print("{} nota(s) de R$ 2.00".format(int(aux / 2)))
aux = int(aux % 2)

print("MOEDAS:")
print("{} moeda(s) de R$ 1.00".format(int(aux)))
moedas = int(moedas % 100)

print("{} moeda(s) de R$ 0.50".format(int(moedas / 50)))
moedas = int(moedas % 50)

print("{} moeda(s) de R$ 0.25".format(int(moedas / 25)))
moedas = int(moedas % 25)

print("{} moeda(s) de R$ 0.10".format(int(moedas / 10)))
moedas = int(moedas % 10)

print("{} moeda(s) de R$ 0.05".format(int(moedas / 5)))
moedas = int(moedas % 5)

print("{} moeda(s) de R$ 0.01".format(int(moedas / 1)))

File: Beecrowd/1021 Notas e Moedas/main.cpp
#include <iostream>

using namespace std;

int main()
{

    int aux;
    double notas;

    cin >> notas;
    int n = notas;
    int moedas = (notas - n) * 100;

    if ((moedas * 1000) % 10 == 9)
    {
        moedas++;
    }

    cout << "NOTAS:" << endl;
    cout << n / 100 << " nota(s) de R$ 100.00" << endl;
    aux = (n % 100);

    cout << aux / 50 << " nota(s) de R$ 50.00" << endl;
    aux = (aux % 50);

    cout << aux / 20 << " nota(s) de R$ 20.00" << endl;
    aux = (aux % 20);

    cout << aux / 10 << " nota(s) de R$ 10.00" << endl;
    aux = (aux % 10);

    cout << aux / 5 << " nota(s) de R$ 5.00" << endl;
    aux = (aux % 5);

    cout << aux / 2 << " nota(s) de R$ 2.00" << endl;
    aux = (aux % 2);

    cout << "MOEDAS:" << endl;
    cout << aux << " moeda(s) de R$ 1.00" << endl;
    moedas = (moedas % 100);

    cout << moedas / 50 << " moeda(s) de R$ 0.50" << endl;
    moedas = (moedas % 50);

    cout << moedas / 25 << " moeda(s) de R$ 0.25" << endl;
    moedas = (moedas % 25);

    cout << moedas / 10 << " moeda(s) de R$ 0.10" << endl;
    moedas = (moedas % 10);

    cout << moedas / 5 << " moeda(s) de R$ 0.05" << endl;
    moedas = (moedas % 5);

    cout << moedas << " moeda(s) de R$ 0.01" << endl;

    return 0;
}